pdns/dnsdistdist/docs/reference/config.rst

```diff
@@ -118,6 +118,10 @@ Control Socket, Console and Webserver
 
   :param str address: An IP address with optional port. By default, the port is 5199.
 
+.. function:: inClientStartup()
+
+  Returns true while the console client is parsing the configuration.
```

Comment:
Doesn't it return `true` also after the client is done parsing the configuration? I think we have something like `g_configurationDone` to indicate whether we are done parsing the configuration.

Reply:
It returns false in the client console. I couldn't imagine a situation in which somebody could call this in the client state after the `>` prompt was shown so I did not think about it.

Reply:
You're right, after the client is done parsing the configuration all commands are passed to the server over the console, so my comment made no sense!

Reply:
pushed something for this